samples/snippets/visualbasic/how-to/conversions/Program.vb

```diff
@@ -14,7 +14,7 @@ Module Program
         Dim MyDateTime As DateTime = DateTime.Parse(MyString)
         Console.WriteLine(MyDateTime)
         ' Displays the following output on a system whose culture is en-US:
-        '       1/1/2009 12:00:00 AM
+        '       1/1/2009 00:00:00
         ' </Snippet1>
```

Comment:
I think (but haven't tested yet) that the output depends on the OS time settings. It can appear in different formats.
Maybe say "Displays output similar to the following:" ?

Reply:
I was basing this on the interactive experience. We can assume that developers will use the interactive C# in the browser... so I want that to at least be consistent.

Reply:
> We can assume that developers will use the interactive C# in the browser.

VB doesn't have interactive experience + I believe I met the "displays output similar to" pattern in many samples before.

Or maybe avoid all that by editing the sample code itself - if appropriate after reviewing the article - to have a consistent output?